src/tests/class_tests/openms/source/MRMAssay_test.cpp

```diff
@@ -376,22 +354,46 @@ START_SECTION(std::vector<OpenMS::AASequence> MRMAssay::addModificationsSequence
   TEST_EQUAL(sequences[8].toString(), String("PEPT(Phospho)DIEK(Oxidation)"));
   TEST_EQUAL(sequences[9].toString(), String("PEPTD(Phospho)IEK(Oxidation)"));
 }
+
 END_SECTION
 
 START_SECTION(std::vector<OpenMS::AASequence> MRMAssay::combineModifications_(OpenMS::AASequence sequence))
```

Comment:
also this fxn could use improved docu, I only understand what it does by reading the test (generally good) but it would be nice if the fxn documentation would also say more clearly what to expect as output